e2e/test.bats

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bats
+
+set -Eeuo pipefail
+
+# This tests that we get the same verification zip path when we run the commands:
+#
+# - config
+# - inspect -build-config -verification
+# - build -verification
+@test "verification output paths match" {
+	rm -rf dist/ out/
+	CONFIG="$(actions-go-build config | grep ZIP_PATH_VERIFICATION | grep -Eo '/.*$')"
+	BUILDENV="$(actions-go-build inspect --build-config --verification | jq -r .Paths.ZipPath)"
+	actions-go-build build -q
+	BUILD="$(actions-go-build build -q -verification -json | jq -r .Config.Paths.ZipPath)"
+
+	echo "$CONFIG"
+	echo "$BUILDENV"
+	echo "$BUILD"
+
+	diff <(echo "$CONFIG") <(echo "$BUILDENV")
+	diff <(echo "$CONFIG") <(echo "$BUILD")
+
+	[[ "$CONFIG" == "$BUILDENV" ]]
+}
```

Comment:
Quick test validating that we get the same verification paths three different ways. This was what the problem looked like initially. It turned out that there was nothing wrong with calculating the paths, but leaving the test here since it proves an important invariant, needed when the CLI is run by the action.yml.